--- streamai-0.1.2/setup.py	2024-03-06 05:19:50.000000000 +0100
+++ streamai-0.1.2.new/setup.py	2024-08-29 16:35:50.000000000 +0200
@@ -2,8 +2,7 @@
 from setuptools import setup, find_packages
 
 directory = os.path.abspath(os.path.dirname(__file__))
-with open(os.path.join(directory, 'README.md'), encoding='utf-8') as f:
-  long_description = f.read()
+long_description = 'na'
 
 setup(name='streamai',
       version='0.1.2',
